### PR TITLE
Fix infinite recursion in dawn bundle build

### PR DESCRIPTION
**Pull request by havogt**
## Technical Description

Fix yoda version with infinite recursion in dawn bundle.

### CODE REVIEW

File: dawn/bundle/CMakeLists.txt [hunk 1]
```diff
@@ -37,9 +37,9 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/../cmake")
 
 include(DawnCloneRepository)
 
-set(DAWN_YODA_GIT_URL "https:///github.com/Meteoswiss-APN/yoda.git"
+set(DAWN_YODA_GIT_URL "https:///github.com/havogt/yoda.git"
     CACHE PATH "URL of the dawn git repository to clone")
-set(DAWN_YODA_GIT_BRANCH "1.0.6" CACHE STRING "Branch of the dawn git repository to clone")
```
Comment (havogt) [inline dawn/bundle/CMakeLists.txt]: probably will be 1.0.9

File: gtclang/bundle/CMakeLists.txt [hunk 2]
```diff
@@ -48,9 +48,9 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/../cmake")
 
 include(GTClangCloneRepository)
 
-set(GTCLANG_YODA_GIT_URL "https:///github.com/MeteoSwiss-APN/yoda.git"
+set(GTCLANG_YODA_GIT_URL "https:///github.com/havogt/yoda.git"
     CACHE PATH "URL of the yoda git repository to clone")
-set(GTCLANG_YODA_GIT_BRANCH "1.0.7" CACHE STRING "Branch of the yoda git repository to clone")
```
Comment (havogt) [inline gtclang/bundle/CMakeLists.txt]: probably will be 1.0.9

File: gtclang/bundle/CMakeLists.txt [hunk 3]
```diff
@@ -90,7 +90,7 @@ yoda_find_package(
   REQUIRED_VARS dawn_DIR
   ADDITIONAL
     DOWNLOAD_DIR ${YODA_DOWNLOAD_DIR}
-    SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../dawn
+    SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../dawn/bundle
```
Comment (havogt) [inline gtclang/bundle/CMakeLists.txt]: this is the fix together with the change in the yoda branch

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
tests passed

**Comment by havogt** [comment]
launch jenkins

File: gtclang/bundle/CMakeLists.txt (hunk 3 above)
Comment (jdahm) [reply gtclang/bundle/CMakeLists.txt]: Looks correct to me!

**Pull request merged**